src/components/BulkEditList/BulkEditListResult/Preview/PreviewAccordion/PreviewAccordion.js

```diff
@@ -29,6 +29,7 @@ const PreviewAccordion = ({ contentData, columnMapping, visibleColumns, isInitia
             contentData={contentData}
             columnMapping={columnMapping}
             visibleColumns={visibleColumnKeys}
+            maxHeight={window.innerHeight * 0.4}
```

Comment:
Please move this coeficient to separate variable, because it's unclear why it's 0.4